QualityControl/docker-compose.dev.yml

```diff
@@ -12,7 +12,9 @@ services:
       - "8080:8080"
     container_name: qcg-application-dev
     volumes:
-      # Mount the current directory to /usr/src/app cached to improve performance
-      - .:/usr/src/app:cached 
+      # Mount the current directory to /usr/src/app
+      - .:/usr/src/app
```

Comment:
As far as I know, the cached option is better for development. What did your research conclude?